client/tasks/fills/purchase.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { registerPlugin } from '@wordpress/plugins';
+import { WooOnboardingTaskListItem } from '@woocommerce/onboarding';
+import { useState, useCallback } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
+import { useSelect } from '@wordpress/data';
+import { ONBOARDING_STORE_NAME, PLUGINS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import CartModal from '../../dashboard/components/cart-modal';
```

Comment:
I believe pre-existing, but just noting some minor styling issues with the modal spacing:
![Screen Shot 2021-09-17 at 3 06 56 PM](https://user-images.githubusercontent.com/10561050/133843640-96a3341b-a893-458d-a460-66949c0aac02.png)

Reply:
Nice catch, I've gone ahead and addressed the gap under the heading in 64841e097b6d439067ac7fa595addc1d55e668a6.